src/sentry/migrations/0878_add_seer_fields_to_group.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 5.1.7 on 2025-04-30 16:28
+
+from django.db import migrations, models
+
+from sentry.new_migrations.migrations import CheckedMigration
+
+
+class Migration(CheckedMigration):
+    # This flag is used to mark that a migration shouldn't be automatically run in production.
+    # This should only be used for operations where it's safe to run the migration after your
+    # code has deployed. So this should not be used for most operations that alter the schema
+    # of a table.
+    # Here are some things that make sense to mark as post deployment:
+    # - Large data migrations. Typically we want these to be run manually so that they can be
+    #   monitored and not block the deploy for a long period of time while they run.
+    # - Adding indexes to large tables. Since this can take a long time, we'd generally prefer to
+    #   run this outside deployments so that we don't block them. Note that while adding an index
+    #   is a schema change, it's completely safe to run the operation after the code has deployed.
+    # Once deployed, run these manually via: https://develop.sentry.dev/database-migrations/#migration-deployment
+
+    is_post_deployment = False
+
+    dependencies = [
+        ("sentry", "0877_integer_drift_release"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="group",
+            name="seer_autofix_last_triggered",
+            field=models.DateTimeField(db_index=True, null=True),
+        ),
+        migrations.AddField(
+            model_name="group",
+            name="seer_is_fixable",
+            field=models.BooleanField(db_default=False, db_index=True, default=False),
```

Comment:
i'm prolly missing context but should there be a `seer_fixability_score` for sorting?

Reply:
I think the decision at the last sync was to just write True/False and use it as a filter, not a sort.